src/Image.php

```diff
@@ -409,32 +422,42 @@
  */
 class Image extends ImageAutodoc implements \ArrayAccess
 {
+
     /**
-     * Set true when we are logging actions.
+     * The logger instance.
      *
-     * @internal 
+     * @var LoggerInterface
      */
-    static private $_enable_logging = false;
+    private static $logger;
```

Comment:
My `phpcs` defaults to `PSR2` (with `phpcs --config-set default_standard PSR2`). I think a note for `phpcs --standard=PSR2 src/` in the readme would be fine.

`phpcs` for the autogenerated enums complains now about: `Opening brace of a class must be on the line after the definition`. Easy fix:
```php
class Access
{
```

`phplint` will only check for syntax errors (such as missing semicolons, forgotten operators, etc.). So for checking coding standards it's not the preferred tool.